src/app/_components/MobileNavigation.tsx

```diff
@@ -0,0 +1,125 @@
+'use client'
+
+import { Fragment, useState, Dispatch, SetStateAction } from 'react'
+
+import Link from 'next/link'
+
+import { List, X } from '@phosphor-icons/react'
+import clsx from 'clsx'
+
+import { Icon } from '@/components/Icon'
+import { Logo } from '@/components/Logo'
+import { SlideOver } from '@/components/SlideOver'
+import { Social } from '@/components/Social'
+import { TextLink } from '@/components/TextLink'
+
+import { PATHS } from '@/constants/paths'
+
+const getInvolvedItems = [PATHS.EVENTS, PATHS.GRANTS]
+const communityItems = [PATHS.ECOSYSTEM, PATHS.GOVERNANCE]
+
+export function MobileNavigation() {
+  const [open, setOpen] = useState(false)
+
+  return (
+    <Fragment>
+      <button
+        className="-mr-4 flex size-12 items-center justify-center lg:hidden"
+        aria-label="Open mobile navigation"
+        onClick={() => setOpen(!open)}
+      >
+        <Icon component={List} color="brand-300" />
+      </button>
```

Comment:
What about refactoring this? Seems like open and close button have the same styles?